pl.suve.colorful.yaml

```diff
@@ -0,0 +1,75 @@
+app-id: pl.suve.colorful
+
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.freepascal
+
+build-options:
+  append-path: /usr/lib/sdk/freepascal/bin
+  env:
+    PPC_CONFIG_PATH: /usr/lib/sdk/freepascal/etc
+    FPCDIR: /usr/lib/sdk/freepascal/fpcsrc
+
+command: colorful
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --device=dri
+
+modules:
+  - name: optipng
+    sources:
+      - type: archive
+        url: https://netcologne.dl.sourceforge.net/project/optipng/OptiPNG/optipng-0.7.8/optipng-0.7.8.tar.gz
+        sha256: 25a3bd68481f21502ccaa0f4c13f84dcf6b20338e4c4e8c51f2cefbd8513398c
+    buildsystem: autotools
+    build-options:
+      config-opts: 
+        - --with-system-libs
+
+  - name: vorbis-tools
```

Comment:
of this is build time only, a cleanup rule of `*` is missing.

Reply:
Fixed in 5659b31a8bc1afbd02b30ac6a324af6067f28c77.